pkgs/applications/office/qownnotes/default.nix

```diff
@@ -1,29 +1,69 @@
-{ mkDerivation, lib, stdenv, fetchurl
-, qmake, qttools, qtbase, qtsvg, qtdeclarative, qtxmlpatterns, qtwebsockets
-, qtx11extras, qtwayland
+{ lib
```

Comment:
And what's the favicon.png added in the commit??

Reply:
Thanks, fixed. Was a mistake.